app/lib/features/chat/widgets/bubble_builder.dart

```diff
@@ -94,6 +120,9 @@ class _ChatBubble extends ConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
+    final replyProfile = ref.watch(chatRoomProvider.notifier).getUserProfile(
+          message.repliedMessage?.author.id ?? '',
+        );
```

Comment:
yeah, why even? If `getUserProfile` accepted `null`, why put a `''` in there? but generally, I'd rewrite that...

```suggestion
    final authorId = message.repliedMessage?.author.id;
    final replyProfile = authorId != null ? ref.watch(chatRoomProvider.notifier).getUserProfile(authorId) : null;
```

Reply:
Yeah I guess it makes sense. Will make changes.